docker/Dockerfile

```diff
@@ -1,17 +1,18 @@
 FROM alpine:edge AS builder
 RUN apk add --no-cache crystal shards libc-dev \
     yaml-dev libxml2-dev sqlite-dev zlib-dev openssl-dev \
-    sqlite-static zlib-static openssl-libs-static
+    yaml-static sqlite-static zlib-static openssl-libs-static
 WORKDIR /invidious
 COPY ./shard.yml ./shard.yml
 RUN shards update && shards install
 RUN apk add --no-cache curl && \
     curl -Lo /etc/apk/keys/omarroth.rsa.pub https://github.com/omarroth/boringssl-alpine/releases/download/1.1.0-r0/omarroth.rsa.pub && \
     curl -Lo boringssl-dev.apk https://github.com/omarroth/boringssl-alpine/releases/download/1.1.0-r0/boringssl-dev-1.1.0-r0.apk && \
     curl -Lo lsquic.apk https://github.com/omarroth/lsquic-alpine/releases/download/2.6.3-r0/lsquic-2.6.3-r0.apk && \
-    tar -xf boringssl-dev.apk && \
-    tar -xf lsquic.apk
-RUN mv ./usr/lib/libcrypto.a ./lib/lsquic/src/lsquic/ext/libcrypto.a && \
+    apk update && apk verify boringssl-dev.apk lsquic.apk
```

Comment:
Do we need to delete some caches after running `apk update`?

Reply:
Oh, and while you're at it.. can you please combine the `RUN apk add` and `RUN tar -xf` layers and remove the apks once they're unpacked?

Reply:
done